gamestonk_terminal/cryptocurrency/overview/coinpaprika_view.py

```diff
@@ -0,0 +1,629 @@
+"""CoinPaprika View"""
+__docformat__ = "numpy"
+
+import argparse
+from typing import List
+from tabulate import tabulate
+from pandas.plotting import register_matplotlib_converters
+from gamestonk_terminal.helper_funcs import parse_known_args_and_warn, check_positive
+import gamestonk_terminal.cryptocurrency.overview.coinpaprika_model as paprika
+from gamestonk_terminal.cryptocurrency.cryptocurrency_helpers import (
+    long_number_format_with_type_check,
+)
+
+
+register_matplotlib_converters()
+
+# pylint: disable=inconsistent-return-statements
+# pylint: disable=C0302, too-many-lines
+
+CURRENCIES = [
+    "BTC",
+    "ETH",
+    "USD",
+    "EUR",
+    "PLN",
+    "KRW",
+    "GBP",
+    "CAD",
+    "JPY",
+    "RUB",
+    "TRY",
+    "NZD",
+    "AUD",
+    "CHF",
+    "UAH",
+    "HKD",
+    "SGD",
+    "NGN",
+    "PHP",
+    "MXN",
+    "BRL",
+    "THB",
+    "CLP",
+    "CNY",
+    "CZK",
+    "DKK",
+    "HUF",
+    "IDR",
+    "ILS",
+    "INR",
+    "MYR",
+    "NOK",
+    "PKR",
+    "SEK",
+    "TWD",
+    "ZAR",
+    "VND",
+    "BOB",
+    "COP",
+    "PEN",
+    "ARS",
+    "ISK",
+]
+PLATFORMS = paprika.get_all_contract_platforms()["platform_id"].tolist()
```

Comment:
We should avoid having these calls not be contained to a controller, or a command even. Same for ones below